docs/en/platform-engineers/terraform.md

```diff
@@ -26,11 +26,13 @@ TEST SUITE: None
 
 By applying Terraform Provider credentials, Terraform controller can be authenticated to deploy and manage cloud resources.
 
-Please refer to [Terraform controller getting started](https://github.com/oam-dev/terraform-controller/blob/master/getting-started.md) on how to apply Provider for Alibaba Cloud or AWS.
+Please refer to [Terraform controller getting started](https://github.com/oam-dev/terraform-controller/blob/master/getting-started.md) on how to apply Provider for Alibaba Cloud, AWS, Azure or GCP.
 
 </details>
 
-## Register `alibaba-rds` Component
+## Alibaba Examples
+
+### Register `alibaba-rds` Component
```

Comment:
This might need to be reverted. Examples are located in https://github.com/oam-dev/kubevela/blob/master/docs/en/end-user/components/cloud-services.md.

Reply:
I have reviewed your link. The examples located here (https://github.com/oam-dev/kubevela/blob/master/docs/en/end-user/components/cloud-services.md) are related to the end-user component. Here are some examples of ComponentDefinition on platform side. This is already present before, i just rename it.

Reply:
Yes. We don't want to emphasize they are `Alibaba Examples`. We used `alibaba-oss` component in the end-user examples, so we have to register the ComponentDefinition here as you said. So L33 should be removed. Thanks.

Reply:
Done. Sorry for my mistake